content/en/docs/Getting Started/pipelines.md

```diff
@@ -121,6 +121,9 @@ both the *Hello World!* and *Goodbye World!* tasks.
     ```bash
     tkn pipelinerun logs hello-goodbye-run -f -n default
     ```
+    
+    *Note*. If you get a message similar to ''tkn: command not found'', please find 
+    installation notes here: https://tekton.dev/docs/cli/
```

Comment:
NIT: Perhaps we could use a link please follow the [*tkn* CLI installation notes](https://tekton.dev/docs/cli/).